dev/src/Command/AddComponentCommand.php

```diff
@@ -111,16 +132,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 ->render();
         }
 
-        $productHomepage = $this->getHelper('question')->ask(
+        $productDocumentation = null;
+        $yamlFileContent = $this->loadYamlConfigContent($new, dirname($proto));
+        if (!empty($yamlFileContent)) {
+            $productDocumentation = $yamlFileContent['publishing']['documentation_uri'] ?? null;
+        }
+        $productDocumentation = $productDocumentation ?? $this->getHelper('question')->ask(
```

Comment:
super-nit, but it'd be _slightly more_ correct to use the `?:` here, as opposed to `??`. the null-coalescing operator `??` is more for when the left hand of the operator doesn't exist. Although it works the same here, it's a little more idiomatic to use `?:` in this case.
```suggestion
        $productDocumentation = $productDocumentation ?: $this->getHelper('question')->ask(
```